doc/user-guide/climada_engine_unsequa.ipynb

```diff
@@ -542,12 +542,12 @@
    "source": [
     "| Attribute | Type | Description |\n",
     "| --- | --- | --- |\n",
-    "| sampling_method | str | The sampling method as defined in [SALib](https://salib.readthedocs.io/en/latest/api.html). Possible choices: 'saltelli', 'fast_sampler', 'latin', 'morris', 'dgsm', 'ff'|\n",
+    "| sampling_method | str | The sampling method as defined in [SALib](https://salib.readthedocs.io/en/latest/api.html). Possible choices: 'saltelli', 'fast_sampler', 'latin', 'morris', 'dgsm', 'ff', 'finite_diff'|\n",
     "| sampling_kwargs | dict | Keyword arguments for the sampling_method. |\n",
     "| n_samples | int | Effective number of samples (number of rows of samples_df)|\n",
     "| param_labels | list(str) | Name of all the uncertainty input parameters|\n",
     "| problem_sa | dict | The description of the uncertainty variables and their distribution as used in [SALib](https://salib.readthedocs.io/en/latest/basics.html). |\n",
-    "| sensitivity_method | str | Sensitivity analysis method from [SALib.analyse](https://salib.readthedocs.io/en/latest/api.html) Possible choices: 'fast', 'rbd_fact', 'morris', 'sobol', 'delta', 'ff'. Note that in Salib, sampling methods and sensitivity analysis methods should be used in specific pairs.|\n",
+    "| sensitivity_method | str | Sensitivity analysis method from [SALib.analyse](https://salib.readthedocs.io/en/latest/api.html) Possible choices: 'sobol', 'fast', 'rbd_fast', 'morris', 'dgsm', 'ff', 'pawn', 'rhdm', 'rsa', 'discrepancy', 'hdmr'. Note that in Salib, sampling methods and sensitivity analysis methods should be used in specific pairs.|\n",
```

Comment:
I do not see it in this PR, but where the allowed pairing options updated in the respective .py files?